SensioLabs/Security/Formatters/JsonFormatter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the SensioLabs Security Checker.
+ *
+ * (c) 2013 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Security\Formatters;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+if (!defined('JSON_PRETTY_PRINT')) {
```

Comment:
this violates psr standards. Including a file containing a class should not define constants.